src/components/FavoriteCities/FavoriteCityForecast/FavoriteCityForecast.jsx

```diff
@@ -25,13 +36,15 @@ function FavoriteCityForecast({ location }: FavoriteCityForecastPropsType): Reac
     setForecastValue();
   }, []);
 
-  const symbolUrl = forecast?.symbol
-    ? `${FORECAST_SYMBOL_LINK}${forecast?.symbol}${FORECAST_SYMBOL_EXT}`
-    : '';
+  const handleFavoriteCityDelete = event => {
+    setFavoriteCities(location, false);
+
+    event.preventDefault();
```

Comment:
What is that for?

Reply:
For removing favorite city forecast from right sidebar.